.buildkite/scripts/install-kind.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+echo "--- Install Kind"
+
+MSG="environment variable missing."
+DEFAULT_HOME="/usr/local"
+KIND_VERSION=${KIND_VERSION:?$MSG}
+HOME=${HOME:?$DEFAULT_HOME}
+KIND_CMD="${HOME}/bin/kind"
+
+if command -v kind
```

Comment:
What that condition here do?

If I understand we check that there is no pre-installed version of Kind that we don't want. How this can be possible?

We update the `PATH` on line `38` so at this stage even if Kind existed in the path, it would not be resolved.

Reply:
We check if `kind` with the corresponding version is already installed. If not, we install it. I plan to pre-install `kind` by building custom VM images for elastic-agent CI. Now we install it on the fly

Reply:
Yes, but my comment I think still stands. 

When this script runs, and let's assume `kind` is already downloaded in the `bin` folder, because the way we define the `PATH` will not locate it under the bin folder, so this `command -v` will always return false. 
Moving `PATH` on the top level, then if its already available under the `bin` folder then it will be resolved and compared.

Reply:
Moved the `PATH` declaration to the top-level script. Thanks @alexsapran for the nice catch!